flask_appbuilder/models/mongoengine/fields.py

```diff
@@ -34,7 +34,8 @@ def process(self, formdata, data=unset_value):
             if marker in formdata:
                 self._should_delete = True
 
-        return super(MongoFileField, self).process(formdata, data)
+        # return super(MongoFileField, self).process(formdata, data)
+        return super().process(formdata, data)
```

Comment:
@shawnzhu could you please include the `return super().process(formdata, data, **kwargs)` to check if it solve the issue? Thank you